src/server/routers/issue.ts

```diff
@@ -5,6 +5,60 @@ import { z } from 'zod';
 import { createProtectedRouter } from '@/server/create-protected-router';
 
 export const issueRouter = createProtectedRouter()
+  .query('infinite', {
+    input: z.object({
+      search: z.string(),
+      limit: z.number().min(1).max(100).default(10),
+      cursor: z.date().nullish(),
+    }),
+    async resolve({ ctx, input: { search, limit, cursor } }) {
+      const issues = await ctx.db.issue.findMany({
+        take: limit + 1,
+        where: {
+          title: {
+            search: search !== '' ? search : undefined,
+          },
+          description: {
+            search: search !== '' ? search : undefined,
+          },
+        },
+        cursor: cursor ? { createdAt: cursor } : undefined,
```

Comment:
Cursor should be the ID and not creatdAt no?

Reply:
Cursor should be a unique and sortable.